src/actions/actions.py

```diff
@@ -1,43 +1,44 @@
-from talon import Context, Module, actions
+from talon import Context, Module, actions, app
 from dataclasses import dataclass
+from ..csv_overrides import init_csv_and_watch_changes
 from .homophones import run_homophones_action
 from .find import run_find_action
 from .call import run_call_action
 
 mod = Module()
 
 ctx = Context()
-ctx.matches = r"""
-tag: user.cursorless
-"""
```

Comment:
Context list not associated with any voice commands are not used. Since cursorless has its own list that are only used by its own voice commands there are no change at all in having these context matches.